deploy/auto-install.sh

```
@@ -190,19 +187,19 @@ upgrade_docker_openwisp() {
 
 	download_docker_openwisp "$openwisp_version"
 
-	cd $INSTALL_PATH &>>$LOG_FILE
+	cd "$INSTALL_PATH" &>>"$LOG_FILE"
 	check_status $? "docker-openwisp download failed."
-	echo $openwisp_version >$INSTALL_PATH/VERSION
+	echo $openwisp_version >"$INSTALL_PATH/VERSION"
 
 	start_step "Configuring docker-openwisp..."
-	for config in $(grep '=' $ENV_BACKUP | cut -f1 -d'='); do
+	for config in $(grep '=' "$ENV_BACKUP" | cut -f1 -d'='); do
```

Shellcheck (0.11.0)

[info] To read lines rather than words, pipe/redirect to a 'while read' loop.

(SC2013)